gloo/test/openssl_utils.cc

```diff
@@ -0,0 +1,43 @@
+#include "gloo/test/openssl_utils.h"
+#if GLOO_HAVE_TRANSPORT_TCP_TLS
+
+#include <cstdlib>
+#include <gloo/common/logging.h>
+
+namespace gloo {
+namespace test {
+
+std::string ca_key_file;
+std::string ca_file;
+std::string pkey_file;
+std::string csr_file;
```

Comment:
didn't see this one in benchmark options, what is this string?

Reply:
deleted